docs/docs/specs/v0/ui-system.md

```diff
@@ -0,0 +1,41 @@
+# UI System
+
+Here is how the UI system is organized into separate concerns:
```

Comment:
we were able to separate the part of the UI system which was going to be more restrictive and "top-down" about how normal UI patterns look - something more akin to Fluent Blocks vs a more open component system like MUI. Because all we need to make our apps are react styled components (and not yet an entire top down system), it seemed like marrying them to the theme name mad more sense. Mosaic is left reserved for such a higher order UI patterns system, and we won't need much more than the styled aurora components to have enough flexibility and power for both labs and product apps

Reply:
I hear the point, but am unconvinced. At least to me mosaic relates to "containerization" -- not the full top down ("blocks") like experience. E.g., Stacks are containers -- hence in my terminology (an in fact in the current concrete implementation) part of mosaic. So this is less about naming than getting more in sync on the layers. And I think the complexity in the layers for me is understanding the current extensibility story -- i.e., how to continue the development of kai within the main application thread.